src/Compilers/Core/Portable/Operations/Loops/ForToLoopOperationInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    internal class ForToLoopOperationUserDefinedInfo
```

Comment:
Any reason this can't just be a struct?

Reply:
> Any reason this can't just be a struct?

It is not common to need this information. The intent is to save some memory for the common case.

---
In reply to: [187115823](https://github.com/dotnet/roslyn/pull/26734#discussion_r187115823) [](ancestors = 187115823)